import-automation/executor/gke/README.md

```diff
@@ -0,0 +1,18 @@
+Deploy to GKE
+
+Import-automation executor can also be deployed to gke. After committing new changes, please do the following to deploy.
+
+1. cd to executor dir.
```

Comment:
Suggestion: let's point to at least the relative path here?

Reply:
Done.